src/sdk/isPegoutQuotePaid.ts

```diff
@@ -9,21 +9,23 @@ const MAX_RETRIES = 3
 const RETRY_DELAY = 3000 // 3 seconds
 
 /**
- * Checks if a pegout quote has been paid by verifying the transaction reported in the pegout status
- * and the OP_RETURN output.
+ * Verifies if a Liquidity Provider Service (LPS) has paid a pegout quote by validating both the transaction
```

Comment:
Just a comment, remember this file is "internal" I'm not sure what the bundler is going to do with the JSDoc of an internal function. My advice is that if you expect the user of the SDK to read this, then add it to the FlyoverSDK class in the wrapper method. If this is documentation for contributors is ok. 

I'm talking specifically about internal functions, the structs that you document and export in the main index of the project will be accessible by the user, no problem there

Reply:
You're right. It's better to place the comments on the visible parts. I just deleted the documentation of this one.